.ci.yaml

```diff
@@ -210,6 +210,15 @@ targets:
     timeout: 90
     scheduler: luci
 
+  - name: Linux linux_host_engine
```

Comment:
nit: Why Linux linux_host_engine, and not Linux host_engine

Reply:
Don't remember the details but there were some clashes in the names with some of the builders in staging. This will be eventually renamed to `Linux Host Engine` when we are ready to move it to prod.